users/richie/home/gui/firefox.nix

```
@@ -0,0 +1,292 @@
+{ pkgs, ... }:
```

If inputs is passed as `specialArgs` I think that this should work.
```suggestion
{ pkgs, inputs, ... }:
```